src/picterra/client.py

```diff
@@ -343,7 +363,7 @@ def edit_detector(
         # Call API and check response
         resp = self.sess.put(
             self._api_url('detectors/%s/' % detector_id),
-            data=body_data
+            json=body_data
```

Comment:
This seems unrelated and I think it's equivalent as far as requests is concerned. I'd chop this change

Reply:
Checked (reverted)